app/views/operational_fields/index.html.erb

```diff
@@ -3,10 +3,10 @@
 
 <header class="block headings-block">
   <div class="inner-block floated-children">
-    <%= render partial: 'shared/heading',
-              locals: { type: 'British fatalities',
-                        heading: "Fields of operation",
-                        big: true } %>
+    <%= render "govuk_publishing_components/components/title", {
+      context: "British fatalities",
+      title: "Fields of operation",
+    } %>
```

Comment:
Would need to check this on integration.
`/government/fields-of-operation/afghanistan/`